atd-vzv/src/views/nav/Header.js

```diff
@@ -119,10 +118,11 @@ const Header = () => {
                 Vision Zero -- Help Austin reach zero traffic deaths
               </h1>
               <img
-                className="vz-logo pl-lg-3"
+                className="vz-logo"
                 // Need to adjust location of public folder to account for /viewer/ basepath
                 src={process.env.PUBLIC_URL + "/vz_logo.svg"}
                 alt="Vision Zero Austin Logo"
+                width={300}
```

Comment:
I noticed a small regression on mobile. I think that the logo could get a bit smaller on mobile to avoid overlap with the hamburger on mobile screens. I used iPhone 12 size in Chrome for the screenshots.

Before 
<img width="390" alt="Screenshot 2023-09-25 at 4 05 59 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/37249039/94c4bd74-c4f7-4ca3-9de1-abc9012888dd">

After
<img width="390" alt="Screenshot 2023-09-25 at 4 06 14 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/37249039/42602b99-d52e-482d-9dec-6aa49de02a40">

Reply:
How does it look now? Had to futz around and push up a few diff changes to test the deploy preview on mobile :p

Reply:
I think that it looks great! ✨